include/GafferSceneUI/Private/Inspector.h

```diff
@@ -337,6 +339,9 @@ class GAFFERSCENEUI_API Inspector::Result : public IECore::RefCounted
 		/// Returns a warning associated with the plug returned
 		/// by `acquireEdit()`. This should be displayed to the user.
 		std::string editWarning() const;
+		/// Returns a user-facing description of the source of the
+		/// fallback value when `SourceType` is `Fallback`.
+		std::string fallbackDescription() const;
```

Comment:
Return `const &` to avoid a compulsory copy?

Reply:
Returning `const &` in da41a2b.